app/views/shared/_user-container.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class='user-container'>
+  <img src="<%= user.photo%>" class='user-img' />
+  <div class='user-div'>
+    <h2><%= link_to user.name, user_url(user) %></h2>
+    <small class='posts_counter'>Number of posts: <%= user.posts_counter %></small>
+  </div>
```

Comment:
- [ ] kindly keep to design wireframe, even after adding user photo in the seed file, I still could not see the picture added, kindly fix
your design :point_down: 
![Screenshot from 2023-08-31 20-24-55](https://github.com/Enoisong/blog_app/assets/66097371/3fef6f0c-f89e-49ed-a64e-67f52ad4d21e)
wireframe :point_down: 
![Screenshot from 2023-07-20 22-19-45](https://github.com/Enoisong/blog_app/assets/66097371/ee9ff3c5-5a18-47e4-8154-1032e8258b8b)